releases/release-1.8/README.md

```diff
@@ -0,0 +1,43 @@
+# Kubeflow 1.8
+
+### Links
+
+- [Release Team](release-team.md)
+- [Calendar](https://arrik.to/kf-release-team-cal)
+- [Meeting Notes](https://arrik.to/kf-release-team-notes)
+- [Recordings](https://arrik.to/kf-release-team-recordings)
+- [Retrospective Document](https://bit.ly/kf-release-1-6-retro)
+- Project Board
+- Contact
+  - [#release](https://app.slack.com/client/T7QLHSH6U/C9V2WT2KV) on slack
+  - [@release-team](https://github.com/orgs/kubeflow/teams/release-team) on GitHub
+
+### TL;DR
+
+The 1.8 release cycle is proposed as follows
+
+- **Friday, May 5th 2023**     : Week 0      - Release team is formed
+- **Monday, May 8th 2023**     : Week 1  C\* - Release cycle begins
+- **Friday, May 12th 2023**    : Week 1  C\* - Roadmaps are finalized
+- **Wednesday, Aug 16th 2023** : Week 15 C\* - Feature freeze
```

Comment:
just curious, what's the reason for increasing feature freeze from typical [11 weeks](https://github.com/kubeflow/community/blob/master/releases/handbook.md#timeline) to 15 weeks?

Reply:
I used [1.7](https://github.com/kubeflow/community/tree/master/releases/release-1.7#tldr) as reference, which used 18 weeks.
The proposal of 15 weeks is the way I found as middle ground between those 18 weeks and dates that could accommodate manifest and distribution testing, plus bug fixing, and be able to get everything on time before Kubecon. This also comes from KFP feedback, as the team would be comfortable having 2.0 by then.
I can definitely change it to what the [handbook](https://github.com/kubeflow/community/blob/master/releases/handbook.md#timeline) says if you think 15 weeks is too much.

Reply:
Thanks for the explanation! I don't have a strong preference, and it makes sense for WG leads to drive the dates if they already have planned features/releases.

Reply:
Let me see if I can find a better middle ground between the standard 11 weeks and the feedback I've received from WG.